crates/wasm/src/keys.rs

```diff
@@ -5,11 +5,39 @@ use wasm_bindgen::prelude::*;
 
 use penumbra_keys::keys::{Bip44Path, SeedPhrase, SpendKey};
 use penumbra_keys::{Address, FullViewingKey};
+use penumbra_proof_params::{
+    DELEGATOR_VOTE_PROOF_PROVING_KEY, NULLIFIER_DERIVATION_PROOF_PROVING_KEY,
+    OUTPUT_PROOF_PROVING_KEY, SPEND_PROOF_PROVING_KEY, SWAPCLAIM_PROOF_PROVING_KEY,
+    SWAP_PROOF_PROVING_KEY, UNDELEGATECLAIM_PROOF_PROVING_KEY,
+};
 use penumbra_proto::{core::keys::v1alpha1 as pb, serializers::bech32str, DomainType};
+use wasm_bindgen_futures::js_sys::Uint8Array;
 
 use crate::error::WasmResult;
 use crate::utils;
 
+#[wasm_bindgen]
```

Comment:
Also, curious, is this an expensive function?

Reply:
I added a description detailing the function's behavior. Since stripping out the proving keys from the wasm binary, calling the `load` method is now required to load the keys in memory at runtime. The extension currently loads in the full **~90MB** wasm binary. Instead, we want the extension to load in the smaller binary without the proving keys, separately download the **.bin** files, and call the `load` function. 

This is an expensive function, but it only needs to be called once for each proving key type (for instance,  initialization of the extension / dapp). This makes the execution traces faster inside the individual web-workers since they don't need to load the entire massive wasm binary every time. 

We don't need to make any changes to `wasm-pack`.

Reply:
I think @hdevalence understands the following point better, but I thought that the idea was that we still want the extension to contain the proving keys, so that it's self contained, we just want to avoid putting them in the WASM itself, which is much more expensive to parse.